rna-interaction-schema_v1.json

```diff
@@ -116,30 +125,34 @@
                             "properties": {
                                 "description": {
                                     "type": "string",
-                                    "description": ""
+                                    "description": "Summary of the "
```

Comment:
Summary of the interaction or partners?

Reply:
Ah thanks, will use the description from the README